HW4.py

```diff
@@ -0,0 +1,16 @@
+# readcsv.py -path_csv example.csv -col_name Marker_Strategy
+
+import argparse
+import csv
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-path_csv')
+parser.add_argument('-col_name')
+ns = parser.parse_args(sys.argv[1:])
+
+result = []
+with open(ns.path_csv) as f:
```

Comment:
The program will crash if the file doesn't exist.
Probably it'd be better to use "try-except" construction.

Reply:
Thank you for your advise :)